src/Propel/Generator/Model/PropelTypes.php

```diff
@@ -193,8 +193,6 @@ class PropelTypes
         self::DATE          => \PDO::PARAM_STR,
         self::TIME          => \PDO::PARAM_STR,
         self::TIMESTAMP     => \PDO::PARAM_STR,
-        self::BU_DATE       => \PDO::PARAM_STR,
-        self::BU_TIMESTAMP  => \PDO::PARAM_STR,
```

Comment:
Why remove these?

Reply:
Those must be duplicate keys if I recall correctly.

Reply:
Stan ftw :)

Reply:
Wow.